Jenkinsfile

```diff
@@ -34,7 +37,7 @@ pipeline {
         axes {
           axis {
             name 'RUSTC_VERSION'
-            values 'stable', 'beta', '1.54.0'
```

Comment:
Removed 1.54 as it's more than 6 months old and some of our dependencies don't compile with that old a version anymore.